### PR TITLE
Submitting lesson 3, list lab exercise.

### PR DESCRIPTION
**Pull request by ptfbanks**
'Eager to get on with the next exercises.

### CODE REVIEW

File: students/ptfbanks/Lesson03/3-2_list_lab.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+print ("Today we are going to be playing with fruits.  It's a healthy exercise.  Don't take it personally")
```
Comment (charlesrobison) [inline students/ptfbanks/Lesson03/3-2_list_lab.py]: print() is a function, so you can remove the space between print and the parentheses.

File: students/ptfbanks/Lesson03/3-2_list_lab.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+add_entry = input("Please add another fruit to lead the list?")
+
+fruits.insert(0,add_entry)
+print (fruits, "\n")
+
+print ("interesting.....  All fruits beginning with 'P' include:")
+p_fruits = []
+for element in fruits:
+    if element[0]== "p":
```
Comment (charlesrobison) [inline students/ptfbanks/Lesson03/3-2_list_lab.py]: Good idea to use lower() here to make it case-insensitive.  i.e., if element[0].lower() == "p":

File: students/ptfbanks/Lesson03/3-2_list_lab.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+for fruit in (like_fruits):
+    a = input(f"Do you like the fruit {fruit} (y/n):")
+    while a != "n" and a != 'y':
+        a = input(f"'Sorry, {a} will not work. Please enter 'y' or 'n' :")
+    if a == 'n':
+        x_fruit.append(fruit)
+print ("fruits you do not care for: ", x_fruit)
+for fruit in (x_fruit):
+    like_fruits.remove(fruit)
+print("That leaves accepable fruits as: ", like_fruits)
```
Comment (charlesrobison) [inline students/ptfbanks/Lesson03/3-2_list_lab.py]: Typo - acceptable

**Comment by ptfbanks** [comment]
1) Thanks for the case-insensitivity tip
3) Typo...I chose math - I never could get English right.

**Pull request merged**